cw721/on-chain-metadata/Cargo.toml

```diff
@@ -41,10 +41,10 @@ optimize = """docker run --rm -v "$(pwd)":/code \
 
 [dependencies]
 cosmwasm-std = "1.0.0-beta"
-cw2 = "0.11"
-cw721 = "0.11"
-cw721-base = { version = "0.11", features = ["library"] }
-schemars = "0.8"
+cw2 = "0.13.4"
+cw721 = "0.13.2"
+cw721-base = { version = "0.13.2", features = ["library"] }
+schemars = "0.8.10"
```

Comment:
Please, check the unresolved comment from the previous review regarding versions with `0.13` vs. `0.13.x`.

Reply:
I did change it but maybe your push reverted them, changing again

Reply:
There was nothing extra on the branch. Otherwise, GitHub wouldn't allow me to push without a `--force` push.

Reply:
Turns out it was for the on-chain metadata and only changed it for `cw20`